lib/storage/noGzipContentTypes.js

```diff
@@ -0,0 +1,7 @@
+// Content types NOT suitable for gzip because
```

Comment:
I guess you created a new file because you plan to use in other places as s3.js. Is that right?

Reply:
Potentially, yes.